src/app/infobox/infobox.component.html

```diff
@@ -1,4 +1,6 @@
 <div *ngIf="this.description" class="card" [hidden]="!isVisible" [style.color]="themeService.cardColor">
+<<<<<<< HEAD
+<<<<<<< HEAD
```

Comment:
It looks like they are some weird merge conflict lines here, please remove them and fix any errors